package.json

```diff
@@ -14,17 +14,32 @@
   "main": "./bin/cli.js",
   "scripts": {
     "lint": "eslint ./lib",
-    "install-commit-validator": "fit-commit-js install"
+    "install-commit-validator": "fit-commit-js install",
+    "test": "jest"
```

Comment:
Doing what @pksjce said we need listed probably on wednesday. Got midterms. @DanielaValero Yes! Will do! Just have to get something testable up first, collecting information so I've got enough knowledge to do this 👍